### ISSUE TITLE
Understanding Conformance refresh

### ISSUE THREAD

**Issue by bruce-usab** [w3c/wcag#5003]
On AG call 17 March, conformance was a topic of discussion and that prompted me to review [Understanding Conformance](https://www.w3.org/WAI/WCAG22/Understanding/conformance).  This doc has served me well over the years, especially the [Understanding Conforming Alternate Version](https://www.w3.org/WAI/WCAG22/Understanding/conformance#conforming-alt-versions) section.  I noticed that the very last line of this section includes a link to itself.
- [ ] delete recursive link

In my review, I also noticed other portions which probably should be updated.
- [ ] [Understanding Conformance Claims](https://www.w3.org/WAI/WCAG22/Understanding/conformance#conformance-claims) includes source code for “a claim enhanced with schema.org metadata”.  The references are to WCAG 2.0 (not 2.2 or 2.1) and includes mention of Windows XP, `longdesc`, and obsolete web browsers.  Is this example still useful?  Could it be deleted?  Is it worth the effort to update?
- [ ] [Examples of Conformance Claims](https://www.w3.org/WAI/WCAG22/Understanding/conformance#examples) the “regular expression” and “boolean logic” bits would be in prose (i.e., not part of the example).  Could/should these examples be numbered?  Should the very dated citations (e.g., WCAG 2.0, XHTML) be updated?
- [ ] There is a space (because of a line break in source) after opening quotation marks.
- [ ] The [reference](https://www.w3.org/WAI/WCAG22/Understanding/conformance#conformance-claims-advisory) to Dublin Core elements has “(future link)” which is seemingly aspirational.  Should this (and five others) be removed?  Should WCAG2-Issues TF actively solicite volunteers?
- [ ] [Understanding Text alternatives](https://www.w3.org/WAI/WCAG22/Understanding/conformance#conformance-claims-advisor) is good as far as it goes, but should it link to [Techniques for Non-text Content](https://www.w3.org/WAI/WCAG22/Understanding/non-text-content#techniques)?

**Comment by bruce-usab** [comment]
@kfranqueiro please see [line 916](https://github.com/w3c/wcag/blob/143028fac8c1727d801e84d6f3f1a8d2b43b2d9e/understanding/conformance.html#L916) of [understanding/conformance.html](../tree/main/understanding/conformance.html) which excerpts the [definition](https://www.w3.org/TR/WCAG22/#dfn-conforming-alternate-versions) (of conforming alternative version) from WCAG2.  That definition links to [Understanding Conformance](https://www.w3.org/WAI/WCAG22/Understanding/conformance) and results in the Understanding doc being self-referential.

Is there a way to correct this in Understanding, or does the fix have to be an erratum?  I scanned the [glossary](https://www.w3.org/TR/WCAG22/#glossary) and I didn’t spot anywhere else this might happen.

**Comment by kfranqueiro** [comment]
> That definition links to [Understanding Conformance](https://www.w3.org/WAI/WCAG22/Understanding/conformance) and results in the Understanding doc being self-referential.

The definition links to a specific heading in the document, though, which seems reasonable and not unlike multiple other links within this page?

A bigger but already-resolved issue is that under Key Terms it currently links to the 2.1 version of the page; this has been fixed in the WCAG 2.2 ED, but it would need to be republished for the fix to manifest here.

I might be able to devise some logic to remove the redundant (and currently wrongly-versioned) URL, but not sure if it's worth the complexity to add.

**Comment by bruce-usab** [comment]
> The definition links to a specific heading in the document, though, which seems reasonable and not unlike multiple other links within this page?

The multiple other in-page links are working as intended.  This link is unlike those others as they each link to a _different_ section (on the [long] page).  This one links to top _of its **own**_ section.  It’s a problem because it  is confusing and disorienting.  One expects to get to new material, but one finds that you’re just re-reading the same content from the few previous paragraphs.

Given that this is the only place this happens, and there is not an easy fix, I agree that it’s not worth the complexity to add.  Also, this bit of recursion has been in place since 2.1 without complaint, and that makes a correction a low priority.

Link to 2.0 (TR space) [Understanding Conforming Alternate Versions](https://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance.html#uc-conforming-alt-versions-head) section (which ends _without_ the recursive link, before “why permit” heading).

For my own reference, here‘s the [diff](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2FUNDERSTANDING-WCAG20%2Fconformance.html&doc2=https%3A%2F%2Fwww.w3.org%2FWAI%2FWCAG22%2FUnderstanding%2Fconformance.html) showing fewer updates to the prose than I had expected!

**Comment by kfranqueiro** [comment]
Apologies Bruce, I had been looking at the wrong definition; I see what you're referring to now (the final paragraph of the "conforming alternate version" definition).

I worry that trying to devise a solution to this in the build system would involve jumping through one or more awkward hoops for the sake of a one-off case. (Would we detect links to the same section within a document as a post-processing pass? How do we know how we should handle them? e.g. in this case removing the entire paragraph makes more sense than unlinking the text. And if we do this, how long until we trip over the build system doing this unexpected clever thing in the future?)

In this instance, the affected link is visibly rendered within a blockquote, so it should be understandable that this link is coming along with the rest of the definition as-is.

Meanwhile, this wouldn't make sense to change in the normative source, since the link makes sense in that context, and I don't think we'd want to start adding provisions within normative text to cater to its inclusion within informative docs.
